### PR TITLE
[Generic] Tags behavior in different apps is wierd

### PR DESCRIPTION
**Pull request by fsh-odoo**
- In  all m2m tags, don't allow the create and edit, just quick create. 
- Add Configuration menu to related Apps (in debug mode)

Task: https://www.odoo.com/web#id=21589&view_type=form&model=project.task&action=333&active_id=646
Pad: https://pad.odoo.com/p/r.YcPQGl6empc9GR6c

### CODE REVIEW

File: addons/fleet/fleet_view.xml [hunk 1]
```diff
@@ -836,6 +836,31 @@
 
         <menuitem action="fleet_vehicle_log_services_act" parent="fleet_vehicles" id="fleet_vehicle_log_services_menu" groups="group_fleet_user"/>
 
+        <record model="ir.ui.view" id="fleet_vehicle_tag_form">
```
Comment (mdi-odoo) [inline addons/fleet/fleet_view.xml]: Place **id** attribute before **model** .

File: addons/fleet/fleet_view.xml [hunk 2]
```diff
@@ -836,6 +836,31 @@
...
+            <field name="arch" type="xml">
+                <form string="Vehicle Tags">
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </form>
+            </field>
+        </record>
+
+        <record model="ir.actions.act_window" id="fleet_vehicle_tag_act">
```
Comment (mdi-odoo) [inline addons/fleet/fleet_view.xml]: Place **id** attribute before **model** .

File: addons/account/views/account_view.xml [hunk 3]
```diff
@@ -400,6 +401,30 @@
            res_model="account.move.line"
            src_model="account.journal"/>
 
+        <!-- Tags -->
+        <record id="view_account_tag_form" model="ir.ui.view">
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Name should be in **<model_name>_view_<view_type>** pattern .

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 4]
```diff
@@ -623,6 +623,38 @@
         action="hr_recruitment_stage_act"
         sequence="1"/>
 
+    <!-- Tag Form View -->
+    <record model="ir.ui.view" id="hr_applicant_category_form">
```
Comment (mdi-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Same as above .

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 5]
```diff
@@ -623,6 +623,38 @@
...
+            <form string="Tags">
+                <group>
+                    <field name="name"/>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <!-- Tag Action -->
+    <record id="open_view_categ_form" model="ir.actions.act_window">
```
Comment (mdi-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Action name should be in **model_name_action** pattern .

File: addons/mass_mailing/views/mass_mailing.xml [hunk 6]
```diff
@@ -694,6 +694,35 @@
             groups="mass_mailing.group_mass_mailing_campaign"
             action="action_view_mass_mailing_stages"/>
 
+        <!--  MASS MAILING CAMPAIGN TAGS !-->
+        <record model="ir.ui.view" id="view_mail_mass_mailing_tag">
```
Comment (mdi-odoo) [inline addons/mass_mailing/views/mass_mailing.xml]: Same as above .

File: addons/note/note_view.xml [hunk 7]
```diff
@@ -36,6 +36,32 @@
...
+        <form string="Tags">
+          <group>
+            <field name="name"/>
+          </group>
+        </form>
+      </field>
+    </record>
+
+    <!-- note Tag Action -->
+    <record model="ir.actions.act_window" id="action_note_tag">
```
Comment (mdi-odoo) [inline addons/note/note_view.xml]: Same as above .

File: addons/note/note_view.xml [hunk 8]
```diff
@@ -36,6 +36,32 @@
         <field name="domain">[('user_id','=',uid)]</field>
     </record>
 
+    <!-- note Tag Form View -->
+    <record model="ir.ui.view" id="view_note_tag_form">
```
Comment (mdi-odoo) [inline addons/note/note_view.xml]: Same as above .

File: addons/account/views/account_view.xml [hunk 9]
```diff
@@ -404,6 +405,30 @@
...
+            <field name="name">Tags</field>
+            <field name="res_model">account.account.tag</field>
+            <field name="view_type">form</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to add a new tag.
+              </p>
+            </field>
+        </record>
+        <menuitem action="account_tag_action" id="menu_action_account_tag_form" parent="account_account_menu" groups="base.group_no_one"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Menu name should be in **model_name_menu** pattern  and also place **id** before all other attributes.

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 10]
```diff
@@ -623,6 +623,38 @@
...
+        <field name="view_type">form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Click to add a new tag.
+            </p>
+        </field>
+    </record>
+
+    <menuitem
+        id="menu_hr_applicant_category"
```
Comment (mdi-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Same as above.

File: addons/note/note_view.xml [hunk 11]
```diff
@@ -169,6 +195,7 @@
     <menuitem name="Notes" id="menu_note_notes" sequence="3" action="note.action_note_note" web_icon="note,static/description/icon.png"/>
     <menuitem name="Configuration" id="menu_note_configuration" sequence="100" parent="menu_note_notes" groups="base.group_no_one"/>
     <menuitem name="Stages" id="menu_notes_stage" parent="menu_note_configuration" action="note.action_note_stage" sequence="21" />
+    <menuitem name="Tags" id="menu_notes_tags" parent="menu_note_configuration" action="note.note_tag_action" sequence="22" />
```
Comment (mdi-odoo) [inline addons/note/note_view.xml]: Same as above .

File: addons/mass_mailing/views/mass_mailing.xml [hunk 12]
```diff
@@ -695,6 +695,35 @@
...
+            <field name="res_model">mail.mass_mailing.tag</field>
+            <field name="view_type">form</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to add a new tag.
+              </p>
+            </field>
+        </record>
+
+        <menuitem name="Campaign Tags" id="menu_view_mass_mailing_tags"
```
Comment (mdi-odoo) [inline addons/mass_mailing/views/mass_mailing.xml]: Same as above .

File: addons/mass_mailing/views/mass_mailing.xml [hunk 13]
```diff
@@ -695,6 +695,32 @@
...
+            <field name="res_model">mail.mass_mailing.tag</field>
+            <field name="view_type">form</field>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to add a new tag.
+              </p>
+            </field>
+        </record>
+
+        <menuitem id="mass_mailing_tag_menu" parent="menu_mass_mailing_configuration" action="mass_mailing_tag_action" sequence="2" groups="mass_mailing.group_mass_mailing_campaign"/>
```
Comment (tivisse) [inline addons/mass_mailing/views/mass_mailing.xml]: group_no_one

File: addons/note/note_view.xml [hunk 14]
```diff
@@ -169,6 +195,7 @@
     <menuitem name="Notes" id="menu_note_notes" sequence="3" action="note.action_note_note" web_icon="note,static/description/icon.png"/>
     <menuitem name="Configuration" id="menu_note_configuration" sequence="100" parent="menu_note_notes" groups="base.group_no_one"/>
     <menuitem name="Stages" id="menu_notes_stage" parent="menu_note_configuration" action="note.action_note_stage" sequence="21" />
+    <menuitem id="notes_tag_menu" parent="menu_note_configuration" action="note_tag_action" sequence="22"/>
```
Comment (mdi-odoo) [inline addons/note/note_view.xml]: Add groups="base.group_no_one" .

**Pull request merged**